scim-server-examples/scim-server-spring-boot/src/main/java/org/apache/directory/scim/example/spring/ScimpleSpringBootApplication.java

```diff
@@ -19,12 +19,25 @@
 
 package org.apache.directory.scim.example.spring;
 
+import org.apache.directory.scim.server.configuration.ServerConfiguration;
+import org.apache.directory.scim.spec.schema.ServiceProviderConfiguration;
 import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
 
-@org.springframework.boot.autoconfigure.SpringBootApplication
-public class SpringBootApplication {
+@SpringBootApplication
+public class ScimpleSpringBootApplication {
 
   public static void main(String[] args) {
-    SpringApplication.run(SpringBootApplication.class, args);
+    SpringApplication.run(ScimpleSpringBootApplication.class, args);
+  }
+
+  @Bean
+  ServerConfiguration serverConfiguration() {
+    return new ServerConfiguration().addAuthenticationSchema(
+      new ServiceProviderConfiguration.AuthenticationSchema()
+      .setType(ServiceProviderConfiguration.AuthenticationSchema.Type.OAUTH_BEARER)
+        .setName(ServiceProviderConfiguration.AuthenticationSchema.Type.OAUTH_BEARER.name())
+        .setDescription("OAuth2 Bearer Token"));
```

Comment:
This configuration should be injected somehow.
NOTE: There is still no auth for this example, this is a placeholder to pass said tests (the actual values could be anything)